code/obj/item/storage/toolbox.dm

```diff
@@ -98,8 +98,22 @@
 	spawn_contents = list(/obj/item/screwdriver,\
 	/obj/item/wirecutters,\
 	/obj/item/device/t_scanner,\
-	/obj/item/crowbar,\
-	/obj/item/cable_coil = 3)
+	/obj/item/crowbar)
+
+	make_my_stuff()
+		if (!istype(src, /obj/item/storage/toolbox/electrical/mechanic_spawn))
+			spawn_contents[ pick(/obj/item/cable_coil,\
```

Comment:
this is some weird syntax, why not just add to the list?

Reply:
There's not enough room in the mechanic one for 3 coils, although we could remove a few of the food items. I think those are largely filler?

Reply:
what? i don't care about the number at all, just the code syntax